src/App.js

```diff
@@ -59,53 +100,30 @@ export default function App() {
     }
   };
 
-  const SideBoardCard = (props) => {
```

Comment:
You separated `SideBoardCard` and `SideBoard` from each other, but they both also have to be outside of `App` (which is a component of it's own).

Just because a component is named `App` doesn't really mean anything, you can name components whatever you want. But you can't (or at least you shouldn't!) have a component defined within another one. You can construct components from other components:
```
<App>
  <MyOtherComponent />
</App>
```
But this is different than actually defining a component.